dashboard/app/controllers/plc/user_course_enrollments_controller.rb

```diff
@@ -28,25 +28,28 @@ def new
   # POST /plc/user_course_enrollments
   # POST /plc/user_course_enrollments.json
   def create
-    user = User.find_by_email_or_hashed_email(user_course_enrollment_params[:user_email])
-    if user.nil?
-      redirect_to action: :new
+    user_emails = user_course_enrollment_params[:user_emails].split /\r\n/
```

Comment:
By default split delimits on any whitespace. Do you intend to restrict it to just \r\n?